pyeapi/eapilib.py

```diff
@@ -415,12 +415,17 @@ def send(self, data):
 
             return decoded
 
-        except (socket.error, ValueError) as exc:
-            if isinstance(exc, socket.error):
+        # socket.error is deprecated in python 3 and replaced with OSError.
```

Comment:
If socket.error and OSError are handled the same, you may want to break them out from ValueError....just for clarity/ease-of-reading for others:

except (socket.error, OSError) as exc:
     ...
except ValueError:
     ...